models/ens/ethereum/ens_ethereum_airdrop_claims.sql

```diff
@@ -31,21 +32,21 @@ SELECT 'ethereum' AS blockchain
 , t.claimant AS recipient
 , t.contract_address
 , t.evt_tx_hash AS tx_hash
-, CAST(t.amount AS DECIMAL(38,0)) AS amount_raw
+, t.amount AS amount_raw
 , CAST(t.amount/POWER(10, 18) AS double) AS amount_original
 , CASE WHEN t.evt_block_time >= (SELECT minute FROM early_price) THEN CAST(pu.price*t.amount/POWER(10, 18) AS double)
     ELSE CAST((SELECT price FROM early_price)*t.amount/POWER(10, 18) AS double)
     END AS amount_usd
-, '{{ens_token_address}}' AS token_address
+, {{ens_token_address}} AS token_address
 , 'ENS' AS token_symbol
 , t.evt_index
 FROM {{ source('ethereumnameservice_ethereum', 'ENSToken_evt_Claim') }} t
 LEFT JOIN {{ ref('prices_usd_forward_fill') }} pu ON pu.blockchain = 'ethereum'
-    AND pu.contract_address='{{ens_token_address}}'
+    AND pu.contract_address= {{ens_token_address}}
     AND pu.minute=date_trunc('minute', t.evt_block_time)
     {% if is_incremental() %}
-    AND pu.minute >= date_trunc("day", now() - interval '1 week')
+    AND pu.minute >= date_trunc('day', now() - interval '7' Day)
     {% endif %}
 {% if is_incremental() %}
```

Comment:
This airdrop is done as far as I can tell so no need to run this incrementally, it can just be a static table.

Reply:
updated!